src/main/java/tools/map/making/MapProperties.java

```diff
@@ -3,51 +3,49 @@
 import java.awt.Color;
 import java.util.List;
 import java.util.Map;
-import java.util.Map.Entry;
 import java.util.TreeMap;
 
 import games.strategy.engine.data.properties.PropertiesUI;
 import games.strategy.triplea.Constants;
 import games.strategy.triplea.image.UnitImageFactory;
-import games.strategy.triplea.ui.mapdata.MapData;
 import games.strategy.util.Tuple;
 
 /**
  * An object to hold all the map.properties values.
  */
-public class MapProperties {
-  public Map<String, Color> colorMap = new TreeMap<>();
-  public String unitsScale = "0.75";
-  public int unitsWidth = UnitImageFactory.DEFAULT_UNIT_ICON_SIZE;
-  public int unitsHeight = UnitImageFactory.DEFAULT_UNIT_ICON_SIZE;
-  public int unitsCounterOffsetWidth = UnitImageFactory.DEFAULT_UNIT_ICON_SIZE / 4;
-  public int unitsCounterOffsetHeight = UnitImageFactory.DEFAULT_UNIT_ICON_SIZE;
-  public int unitsStackSize = 0;
-  public int mapWidth = 256;
-  public int mapHeight = 256;
-  public boolean mapScrollWrapX = true;
-  public boolean mapScrollWrapY = false;
-  public boolean mapHasRelief = true;
-  public int mapCursorHotspotX = 0;
-  public int mapCursorHotspotY = 0;
-  public boolean mapShowCapitolMarkers = true;
-  public boolean mapUseTerritoryEffectMarkers = false;
-  public boolean mapShowTerritoryNames = true;
-  public boolean mapShowResources = true;
-  public boolean mapShowComments = true;
-  public boolean mapShowSeaZoneNames = false;
-  public boolean mapDrawNamesFromTopLeft = false;
-  public boolean mapUseNationConvoyFlags = false;
-  public String dontDrawTerritoryNames = "";
-  public boolean mapMapBlends = false;
+class MapProperties {
+  private Map<String, Color> colorMap = new TreeMap<>();
```

Comment:
IIRC, the fields of this class are accessed reflectively.  I don't remember if the reflection code handled accessibility correctly.  You might want to run through some of the map making tools to ensure the code paths that use this type are running as expected.

Reply:
I'm simply reverting all the changes in this class.